apps/package.json

```diff
@@ -21,7 +21,6 @@
   "browserify": {
     "transform": [
       "require-globify",
-      "envify",
```

Comment:
No need to specify this in configuration if we're defining it inline in the build command.

Since we're no longer passing a custom configuration we might want to define it here _instead_ of the build command, but I'm not sure what that does to the ordering of the transforms, and we're currently doing envify after babelify.